### PR TITLE
Changed how OrmStoreWebSQL behaves

### PR DESCRIPTION
**Pull request by TheCodeDestroyer**
Changed the store so it will work like we talked about.

### CODE REVIEW

**Comment by TheCodeDestroyer** [comment]
@alexweber I can't assign you but at least I can mention you :)

File: src/sqlite_dialect.js [hunk 1]
```diff
@@ -8,14 +8,14 @@ class SqliteDialect {
...
-            sql = `CREATE TABLE IF NOT EXISTS \` ${tableName} + \` (`,
+            sql = 'CREATE TABLE IF NOT EXISTS `' + tableName + '` (',
             columnDefinitions = [];
         for(let i = 0; i < columns.length; i++) {
             let column = columns[i],
                 columnName = column[0],
                 columnType = typeMapper.columnType(column[1]),
                 columnInfo = column[2] ? ` ${column[2]}`  : '';
-            columnDefinitions.push(`\`${columnName}\` ${columnType}  ${columnInfo}`);
+            columnDefinitions.push('`' + columnName + '` ' + columnType + ' ' + columnInfo);
```
Comment (alexweber) [inline src/sqlite_dialect.js]: Hey @TheCodeDestroyer, why you go back to ES5 on these?

File: src/sqlite_dialect.js [hunk 2]
```diff
@@ -8,14 +8,14 @@ class SqliteDialect {
     // [['id', 'VARCHAR(32)', 'PRIMARY KEY'], ['name', 'TEXT']]
     createTable(tableName, columns) {
         let typeMapper = this.typeMapper,
-            sql = `CREATE TABLE IF NOT EXISTS \` ${tableName} + \` (`,
+            sql = 'CREATE TABLE IF NOT EXISTS `' + tableName + '` (',
```
Comment (alexweber) [inline src/sqlite_dialect.js]: Y U NO ES6?

File: src/sqlite_dialect.js (hunk 1 above)
Comment (TheCodeDestroyer) [inline src/sqlite_dialect.js]: @alexweber I wasn't sure if Babel(6to5) supports template escaping, I tried it out on their page and it didn't work, tried it again yesterday and it works maybe I was doing something wrong...
I'll change it back...

Comment (alexweber) [inline src/sqlite_dialect.js]: @TheCodeDestroyer It works! :)

**Comment by TheCodeDestroyer** [comment]
@alexweber Fixed back to ES6 templates

**Comment by alexweber** [comment]
@TheCodeDestroyer :+1:

**Comment by alexweber** [comment]
@TheCodeDestroyer Merging this like we talked about!

**Pull request merged**